pkg/etcd/init/init.go

```diff
@@ -0,0 +1,121 @@
+package init
```

Comment:
There are a couple of variables in here that I think could be moved to the `defaults.go` file. See [1]